flexmeasures/data/models/planning/__init__.py

```diff
@@ -54,6 +60,7 @@ def __init__(
         start,
         end,
         resolution,
+        asset: Optional[Asset] = None,
```

Comment:
Doesn't sensor become optional? And should we raise if both sensor and asset are passed, and also if neither is passed?

Reply:
You're right! Now it will raise if both are None (`WrongEntityException`)